mmv1/third_party/terraform/acctest/bootstrap_test_utils.go.tmpl

```diff
@@ -34,6 +34,7 @@ import (
 	"google.golang.org/api/servicenetworking/v1"
 	"google.golang.org/api/serviceusage/v1"
 	sqladmin "google.golang.org/api/sqladmin/v1beta4"
+	"cloud.google.com/go/bigquery"
```

Comment:
we have a guide on how to update dependencies to include this. You'll need to update the correct go.mod and go.sum at least. https://googlecloudplatform.github.io/magic-modules/develop/update-dependencies/